src/sentry/conf/server.py

```diff
@@ -1757,12 +1757,6 @@ def build_cdc_postgres_init_db_volume(settings):
                 "max_wal_senders=1",
             ],
             "entrypoint": "/cdc/postgres-entrypoint.sh" if settings.SENTRY_USE_CDC_DEV else None,
-            "healthcheck": {
```

Comment:
This seems to undo the work in #21741 

Was it unnecessary all along?

Would also need to remove some of the code that was added on that PR? (e.g. `get_docker_low_level_client`)

Reply:
Oh, good pointing that out, yeah, all that code should be removed. Didn't think about that.

Reply:
There's still value for healthchecks (like in #21737), just that we shouldn't have docker run them (see PR description). I'm gonna add a healthcheck script for setup-sentry to loop on for now.